runtime/calamari/src/weights/pallet_parachain_staking.rs

```diff
@@ -407,323 +348,296 @@ impl<T: frame_system::Config> pallet_parachain_staking::WeightInfo for Substrate
     // Storage: ParachainStaking Total (r:1 w:0)
     // Storage: ParachainStaking AwardedPts (r:2 w:1)
     // Storage: ParachainStaking AtStake (r:1 w:10)
-    // Storage: MoonbeamOrbiters OrbiterPerRound (r:1 w:0)
-    // Storage: MoonbeamOrbiters CurrentRound (r:0 w:1)
+    // Storage: System Account (r:101 w:101)
     // Storage: ParachainStaking SelectedCandidates (r:0 w:1)
     // Storage: ParachainStaking DelayedPayouts (r:0 w:1)
-	#[rustfmt::skip]
-    fn round_transition_on_initialize(x: u32, y: u32, ) -> Weight {
-		(5_114_000 as Weight)
-			// Standard Error: 796_000
-			.saturating_add((52_172_000 as Weight).saturating_mul(x as Weight))
-			// Standard Error: 2_000
-			.saturating_add((156_000 as Weight).saturating_mul(y as Weight))
-			.saturating_add(T::DbWeight::get().reads(195 as Weight))
-			.saturating_add(T::DbWeight::get().reads((3 as Weight).saturating_mul(x as Weight)))
-			.saturating_add(T::DbWeight::get().writes(188 as Weight))
-			.saturating_add(T::DbWeight::get().writes((1 as Weight).saturating_mul(x as Weight)))
-	}
+    fn round_transition_on_initialize(x: u32, _y: u32, ) -> Weight {
+        (1_631_970_000 as Weight)
```

Comment:
doublecheck

Reply:
This means a round change takes ~2ms, which when compared to the old pallet_session::rotate_session() mechanism, which used the full block is a good net gain.
Also, while the constant part is more expensive than moonbeam, the part that scales with `x` is much cheaper - this seems to be a benchmarking difference.